source/DontTouchMe.hx

```diff
@@ -0,0 +1,116 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.math.FlxPoint;
+import flixel.util.FlxColor;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+
+/**
+ * ...
+ * @author Keith Cissell
+ */
+class DontTouchMe extends Enemy 
+{
+	private var dtmPosition:FlxPoint;
+	private var xSpeed:Float = -30;
+		
+	// Hitboxes
+	public var takeDamageBox:FlxObject;
+	private var tdbXoffset:Float = 5;
+	private var tdbYoffset:Float = 0;
+	public var giveDamageBox:FlxObject;
+	private var gdbXoffset:Float = 2;
+	private var gdbYoffset:Float = 5;
+	
+		
+	/**
+	 * Intializer
+	 *
+	 * @param	X	Starting x coordinate
+	 * @param	Y	Starting y coordinate
+	 */
+	public function new(?X:Float = 0, ?Y:Float = 0) 
+	{
+		super(X, Y, AssetPaths.DontTouchMe__png);
+				
+		name = "DontTouchMe";
+		dtmPosition = getPosition();
+		
+		// Initialize gravity. Assumes the currentState has GRAVITY property.
+		acceleration.y = (cast FlxG.state).GRAVITY;
+		maxVelocity.y = acceleration.y;
+		
+		// Initialize X movement
+		velocity.x = xSpeed;
+		
+		// Set hitboxes
+		takeDamageBox = new FlxObject((X + tdbXoffset), (Y + tdbYoffset), 22, 5);
+		giveDamageBox = new FlxObject((X + gdbXoffset), (Y + gdbYoffset), 30, 27);
```

Comment:
I think that the width/height of the boxes should be parameters too, if that makes sense.

Side note, I think at some point in the future, we may change the bad guy to having a single hitbox, since we're adding extra ones to the player for stomping/block-hitting detection. No need to change that stuff now, though.